schema/v1.json

```diff
@@ -364,6 +365,12 @@
      "default":"",
      "minLength": 16
     },
+   "coreos-assembler.variant": {
+     "$id":"#/properties/coreos-assembler.variant",
```

Comment:
My bikeshed for this is `coreos-assembler.config-variant` to be consistent with other `coreos-assembler.config-*` keys.